src/com/base_java/webapp/storage/AbstractArrayStorage.java

```diff
@@ -0,0 +1,72 @@
+package com.base_java.webapp.storage;
+
+import com.base_java.webapp.model.Resume;
+
+import java.util.Arrays;
+import java.util.logging.Logger;
+
+public abstract class AbstractArrayStorage implements Storage {
+    protected static final int RESUME_NOT_FOUND = -1;
+    protected static final int FLAG_FOUNDED = 0;
+    protected static Logger logger = Logger.getLogger(ArrayStorage.class.getName());
+    protected static final int MAX_LENGTH = 10000;
+    public Resume[] resumes = new Resume[MAX_LENGTH];
+    protected static int size = 0;
+
+    public int sizeOfArray() {
+        return size;
+    }
+
+    public void update(Resume resumeForUpd) {
+        int index = getIndex(resumeForUpd.getId());
+        if (index > FLAG_FOUNDED) {
+            resumes[index].setFullName(resumeForUpd.getFullName());
+            logger.info("Resume updated.");
+        } else logger.info("Not found.");
+    }
+
+    public Resume getByID(Integer ID) {
```

Comment:
Вопрос чисто эстетики, но по названию метода и названию входного параметра, понятно, что делает метод.
Например:
```suggestion
    public Resume get(int id);
```
Методы/переменные именуются по правилу camel case.